python_modules/dagster/dagster/_core/definitions/assets.py

```diff
@@ -69,6 +72,19 @@
 ASSET_SUBSET_INPUT_PREFIX = "__subset_input__"
 
 
+@whitelist_for_serdes
```

Comment:
I'm hesitant to commit to storage types that we need to support forever, before we've learned more about what the ontology should look like. There's also the question of `UserAssetOwner` vs. `UserOwner` vs. `User` vs. `Person`. Thoughts on just storing a string for now?

Reply:
Yeah that makes sense, we can remove the `whitelist_for_serdes` tags here.

I'd prefer on the backend to still have `UserAssetOwner` and `TeamAssetOwner` though so we don't need to check for each string whether its a user, team, or if the `team:` prefix is inside. But these objects we can change whenever we want.

Reply:
👍